contents/blog/why-companies-self-host.md

```diff
@@ -0,0 +1,80 @@
+---
+date: 2022-01-14
+title: The 3 critical reasons companies choose self-hosted analytics
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+keywords: ["self host analytics","self hosted analytics"]
+featuredImage: ../images/blog/hipaa-compliant-product-analytics.png
```

Comment:
This is a great picture, but it's an old-brand hedgehog. We're using a new style now, so this will need to be changed. 

There are some generic images in the new style already uploaded and available, e.g. contents/images/blog/posthog-company-culture-blog.png and contents/images/blog/posthog-blog-image.png , or can request a new one of @lottiecoxon if it needs custom, new art.